core/src/main/java/io/grpc/CallOptions.java

```diff
@@ -86,6 +91,23 @@ public Long getDeadlineNanoTime() {
     return deadlineNanoTime;
   }
 
+  /**
+   * Retruns the Codec used for compression and decompression, or {@code null} if none is set.
```

Comment:
As this is a work in progress, it will likely change.  This is just *something* that works now.